templates/janitor/3/rancher-compose.yml

```diff
@@ -34,3 +34,20 @@
       default: "**None**"
       required: false
       type: "string"
```

Comment:
Typo errors in variable definitions DEBUG and LOOP. Missed '"' chars in labels and description.  Otherwise, variables need to be required: true (with default values) in other pass a value to docker env variables.

Please, change your variable definitions with that. 
```
    - variable: "DEBUG"
      label: "Enable more debugging output on pattern matches"
      description: "Read https://github.com/meltwater/docker-cleanup#environment-variables"
      required: true
      default: 0
      type: "enum"
      options:
        - 0
        - 1
    - variable: "LOOP"
      label: "Ability to do non-looped cleanups, run it once and exit. Defaults to yes to run it forever in loops."
      description: "Read https://github.com/meltwater/docker-cleanup#environment-variables"
      required: true
      default: true
      type: "enum"
      options:
        - false
        - true
```